code/__DEFINES/misc.dm

```diff
@@ -564,4 +564,6 @@ var/global/list/ghost_others_options = list(GHOST_OTHERS_SIMPLE, GHOST_OTHERS_DE
 
 #define NUKE_NEAR_MISS 1
 #define NUKE_MISS_STATION 2
-#define NUKE_SYNDICATE_BASE 3
\ No newline at end of file
+#define NUKE_SYNDICATE_BASE 3
+
+#define SNAKE_SPAM_TICKS 600 //how long between cardboard box openings that trigger the '!'
```

Comment:
I feel this doesn't really need to be a define when it's used once in a single file.